staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode_test.go

```diff
@@ -105,13 +105,13 @@ func TestDecode(t *testing.T) {
 			assertOnError: assertNilError,
 		},
 	} {
-		ms := tc.modes
-		if len(ms) == 0 {
-			ms = allDecModes
+		decModes := tc.modes
```

Comment:
just can't bring yourself to go to decodeModes? :)

Reply:
Hold on, are you telling me "dec" is an abbreviation of "decode"?

Reply:
(I went with the type name https://pkg.go.dev/github.com/fxamacker/cbor/v2#DecMode here).